README.md

```diff
@@ -9,7 +9,7 @@ LoopBack makes it easy to build modern applications that require complex integra
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-# Status: ALPHA
+# Status: BETA
```

Comment:
This should not change yet. We're not going to beta just yet, beta implies that the API has solidified, and that we're running up to release in a relatively short time frame.